File: tests/GeneratedProject/Site.fs
namespace SynthUI

module Site =
    open WebSharper
    open WebSharper.Sitelets
    open WebSharper.Sitelets.Tests.Server


    [<Website>]
    let Main =
        Application.SinglePage (fun ctx ->
                let HomePage (ctx: Context<_>) =
                    Content.Page(
                        Title = "WebSharper tests",
                        Body = [
                            Elt("h1", Text "WebSharper tests")
                            ]
                            )
                HomePage ctx
            )

File: tests/Web/Site.cs
using System;
using System.Threading.Tasks;
using WebSharper;
using WebSharper.Sitelets;
using C = WebSharper.Sitelets.Tests.Client;

namespace Web
{
    public class ExtraBundleTest
    {
        [JavaScript]
        public static C.Node SayHello()
        {
            Console.WriteLine("Hello world from System.Console!");
            WebSharper.JavaScript.Console.Log("Hello world from WebSharper.JavaScript.Console!");
            return C.Elt("div", C.Text("Hello from an inline control!"));
        }

        public static Task<Content> MainPage(Context<object> ctx, object endpoint) =>
            Content.Page(
                new WebSharper.Web.InlineControl(() => SayHello()),
                Bundle: "extrabundle"
            );
    }
}